api/src/queries/events.ts

```diff
@@ -111,12 +111,63 @@ export const eventsAggregations = {
       field: 'aggregatableValues.isAvailableOnline',
     },
   },
-  // https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-daterange-aggregation.html
-  // https://github.com/wellcomecollection/content-api/issues/220
   timespan: {
-    terms: {
-      size: 5,
-      field: 'filter.timespan', // use filter values and not create aggregations for it
+    nested: {
+      path: 'filter.times',
+    },
+    aggs: {
+      all: {
+        filter: {
+          bool: {
+            filter: [],
+          },
+        },
+        aggs: {
+          count_parent: {
+            reverse_nested: {},
+          },
+        },
+      },
+      past: {
+        filter: {
+          bool: {
+            filter: [
+              {
+                range: {
+                  'filter.times.endDateTime': {
+                    lt: 'now',
+                  },
+                },
+              },
+            ],
+          },
+        },
```

Comment:
The attraction towards using a range filter here was that we could re-use our `getTimespanRange` [helper function](https://github.com/wellcomecollection/content-api/blob/main/api/src/controllers/utils.ts#L102), whereas a [range aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-range-aggregation.html) used a different formatting/language (to/from)

Reply:
I'll be replacing this code to use the helper after we chat, I felt it was more readable like this for now.